content/en/docs/setup/platform-setup/gke/index.md

```diff
@@ -79,3 +79,32 @@ Follow these instructions to prepare a GKE cluster for Istio.
         --clusterrole=cluster-admin \
         --user=$(gcloud config get-value core/account)
     {{< /text >}}
+
+## Multi-cluster communication
+
+In some cases, a firewall rule must be explicitly created to allow cross-cluster traffic
```

Comment:
To mimic the format of the rest of the document, you need a period after this sentence as well as the two numbered sentences.

Reply:
Done, thanks!